integration/oneup/Tiltfile

```diff
@@ -1,4 +1,10 @@
 # -*- mode: Python -*-
 
-docker_build('gcr.io/windmill-test-containers/integration/oneup', '.')
+# If you get push errors, you can change the default_registry.
+# Create tilt_option.json with contents: {"default_registry": "gcr.io/my-personal-project"}
+# (with your registry inserted). tilt_option.json is gitignore'd, unlike Tiltfile
+default_registry(read_json('tilt_option.json', {})
+                 .get('default_registry', 'gcr.io/windmill-public-containers/servantes'))
```

Comment:
fwiw, we don't give our CI access to windmill-public-containers, only windmill-test-containers